modules/apps/notification/notification-api/src/main/java/com/liferay/notification/util/NotificationRecipientSettingUtil.java

```diff
@@ -27,6 +29,14 @@ public static Map<String, Object> toMap(
 
 			Object value = notificationRecipientSetting.getValue();
 
+			if (StringUtil.equals(
+					notificationRecipientSetting.getName(),
+					"singleRecipient")) {
+
+				value = GetterUtil.getBoolean(
+					notificationRecipientSetting.getValue());
+			}
+
 			if (Validator.isXml(notificationRecipientSetting.getValue())) {
```

Comment:
Hey @seltong 

Could you add `else` in this `if`, please?

Reply:
yeah, good noted